README.md

```diff
@@ -81,6 +79,80 @@ def create_new_bitbucket_creds_flow():
 create_new_bitbucket_creds_flow()
 ```
 
+#### Create a BitBucketRepository block for a public repo.
```

Comment:
Would you mind sharing how the documentation right sidebar looks with a period in the section header?

Reply:
For sure

<img width="193" alt="Screen Shot 2022-11-29 at 2 55 55 PM" src="https://user-images.githubusercontent.com/42048900/204645841-6b229dce-7504-46b6-937b-960b76ee1efa.png">

Reply:
I guess the periods look fine, but if we do use periods, maybe Write and run a flow can use one too.

Reply:
@ahuang11 I personally think no period looks better, but what do you think?

Reply:
I don't have a strong preference, but if we go with precedent in other collections, then no periods.